### PR TITLE
Fix price range bug

### LINKED ISSUE #201: [PE-D] Find command does not seem to work with start price and end price
Tried:
`find sp:0 ep:10`
`find n:test sp:0`

Fails this specification:

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: Spyobird/ped#8

### LINKED ISSUE #204: [PE-D] No hint for what is defined as same product in add
It is a bit confusing when using the app and displayed 'product already exists
It turned out the criteria is when the name and the category are the same
but this is not reflected in the user guide.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: ddx-510/ped#5

### PR DESCRIPTION
**Pull request by mazx4960**
# Changes
- Fix price range bug
- Specify what same product means in documentation

# Related Issues/ PRs
- Fixes #204 
- Fixes #201

### CODE REVIEW

File: src/main/java/seedu/ibook/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -160,6 +162,22 @@ public static Price parsePrice(String price) throws ParseException {
         return new Price(trimmedPrice);
     }
 
+    /**
+     * Parses a {@code String price} into a {@code Price}.
```
Comment (MechFroG88) [inline src/main/java/seedu/ibook/logic/parser/ParserUtil.java:166]: I think you forget to change the javadocs comment

File: src/main/java/seedu/ibook/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -160,6 +162,22 @@ public static Price parsePrice(String price) throws ParseException {
...
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code price} is invalid.
+     */
+    public static PriceRange parsePriceRange(String startPrice, String endPrice) throws ParseException {
+        requireAllNonNull(startPrice, endPrice);
+        String trimmedStartPrice = strip(startPrice);
+        String trimmedEndPrice = strip(endPrice);
+        if (!PriceRange.isValidPriceRange(trimmedStartPrice, trimmedEndPrice)) {
+            throw new ParseException(PriceRange.MESSAGE_CONSTRAINTS);
```
Comment (MechFroG88) [inline src/main/java/seedu/ibook/logic/parser/ParserUtil.java:176]: I think the error

`Start price should be less than or equal to end price`

will be thrown when the price is not in a valid format.

Might be caught as a bug

Comment (mazx4960) [reply src/main/java/seedu/ibook/logic/parser/ParserUtil.java:176]: Yeap thanks for spotting this!

**Review by MechFroG88 — APPROVED** [review]
LGTM

**Pull request merged**